nslsii/__init__.py

```diff
@@ -92,7 +93,9 @@ def configure_base(
         True by default. Console output and exception stack traces will be
         written to IPython log file when IPython logging is enabled.
     publish_documents_to_kafka: boolean, optional
-        False by default. If True publish bluesky documents to a Kafka message broker.
+        False by default. If True publish bluesky documents to a Kafka message broker
+    tb_minimize : boolean, optional
```

Comment:
```suggestion
    tb_minimize: boolean, optional
```

Reply:
numpydoc says there should be a space before the `:`.

Reply:
Hm, it's conflicting with what we discovered recently with @danielballan. Dan, do you remember where we saw issues with the latest Sphinx?